spec/support/raise_on_js_errors.rb

```diff
@@ -14,7 +14,7 @@
     if js_errors.present?
       exception_headline = "This test caused JS errors."
       exception_details = js_errors.map(&:message).map { |line| line.indent(2, " ") }.join("\n")
-      raise JavaScriptError, exception_headline + "\n" + exception_details
+      raise JavaScriptError, [exception_headline, exception_details].join("\n")
```

Comment:
Style/StringConcatenation: Prefer string interpolation to string concatenation. (https://rubystyle.guide#string-interpolation)